src/builds/python/mac/build_native.py

```diff
@@ -0,0 +1,196 @@
+import os
+import sys
+import shutil
+import subprocess
+import platform
+
+def parse_arguments(args):
+    options = {
+        'nointerop': False,
+        'nopython': False,
+        'noregen': False,
```

Comment:
Naming of these flags can be a little confusing:
- Calling `build` script in `python` folder has a flag `nopython`. A little bit weird at first
- 'noregen', not regenerating what?

Possible improvement:
- `nopython` -> `nocopy` / `compile-only`
- `noregen` -> `noprojects` / `nointeropprojects`
-  Use `skip` instead of `no` (ex. `skip-interop`, `skip-projects`...)

Reply:
left a comment on the other one (https://github.com/quixio/quix-streams/pull/87#discussion_r1155476816). I'll leave that unresolved, so you can read & see the modifications.